#TRY & EXCEPT              21/06/24
try:
    print(x)
except NameError:
    print('please')
finally:
    print('hello')
    